nevergrad/functions/multiobjective/core.py

```diff
@@ -45,17 +53,23 @@ def compute_aggregate_loss(self, losses: ArrayLike, *args: Any, **kwargs: Any) -
         """Given parameters and the multiobjective loss, this computes the hypervolume
         and update the state of the function with new points if it belongs to the pareto front
         """
-        # We compute the hypervolume
+        if self._auto_bound > 0:
+            self._upper_bounds = np.array(losses) if self._auto_bound == self._bound_budget else np.maximum(self._upper_bounds, np.array(losses))
+            self._lower_bounds = np.array(losses) if self._auto_bound == self._bound_budget else np.minimum(self._lower_bounds, np.array(losses))
+            self._auto_bound -= 1
+            if self._auto_bound == 0:
+                self._upper_bounds = self._upper_bounds + 1. * (self._upper_bounds - self._lower_bounds)
```

Comment:
why the additional space? that seems dangereous to me (possibly much worst points could get well better loss than current points)